### ISSUE TITLE
Future...

### ISSUE THREAD

**Issue by BjornFJohansson** [mengqvist/DNApy#46]
Hi, does DNApy have a future? Is there a beta release on the horizon perhaps? I am using ApE a lot, but I dont really like tcl. DNApy could perhaps be an alternative?

**Comment by mengqvist** [comment]
Yes, the idea is for DNApy to be an alternative to ApE with similar functionality. I would very much like for there to be a future, but I just can't seem to make time for developing it. I do have a development branch where I have been changing a lot of the plumbing regarding how sequences are represented and operated on. When these changes will be connected back to the GUI is unclear. I was also thinking of moving the whole GUI to Tk instead of using wxWidgets. But to answer your question, the future of DNApy looks quite bleak at the moment. No releases on the horizon. :(
